Gems/Atom/RHI/Metal/Code/Source/RHI/BufferView.cpp

```diff
@@ -61,7 +61,9 @@ namespace AZ
                                                                          offset : m_memoryView.GetOffset()
                                                                     bytesPerRow : (viewDescriptor.m_elementCount * bytesPerPixel)];
                 AZ_Assert(mtlTexture, "Failed to create texture");
-
+                [mtlTextureDesc release];
```

Comment:
I think I had to remove this recently due to double release crash. hopefully this will get corrected in a future PR.